demisto_sdk/commands/download/downloader.py

```diff
@@ -300,7 +309,7 @@ def build_custom_content(self) -> None:
         Build a data structure called pack content that holds basic data for each content entity instances downloaded from Demisto.
         For example check out the CUSTOM_CONTENT variable in downloader_test.py
         """
-        custom_content_objects: list = self.all_custom_content_objects if self.all_custom_content_objects else\
+        custom_content_objects = self.all_custom_content_objects if self.all_custom_content_objects else\
```

Comment:
why did you change this? whats returned from the func?

Reply:
both values are lists, why removed?

Reply:
because you don't need this annotation, mypy knows.